samples/interactive-tutorials/src/main/java/search/SearchSimpleQuery.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * [START retail_search_for_products_with_query_parameter]
+ * Call Retail API to search for a products in a catalog
+ * using only search query.
+ */
+
+package search;
+
+import com.google.cloud.retail.v2.SearchRequest;
+import com.google.cloud.retail.v2.SearchResponse;
+import com.google.cloud.retail.v2.SearchServiceClient;
+import java.io.IOException;
+import java.util.UUID;
+
+public final class SearchSimpleQuery {
+
+  /** This variable describes project number getting from environment variable. */
+  private static final String YOUR_PROJECT_NUMBER = System.getenv("PROJECT_NUMBER");
+
+  /** This variable describes default catalog name. */
+  private static final String DEFAULT_CATALOG_NAME =
+      String.format("projects/%s/locations/global/catalogs/default_catalog", YOUR_PROJECT_NUMBER);
+
+  /**
+   * This variable describes default search placement name. Using for identify the Serving Config
+   * name.
+   */
+  private static final String DEFAULT_SEARCH_PLACEMENT_NAME =
+      DEFAULT_CATALOG_NAME + "/placements/default_search";
+
+  /** This variable describes a unique identifier to track visitors. */
+  private static final String VISITOR_ID = UUID.randomUUID().toString();
```

Comment:
This class should begin with a main function, which provides examples for the user to override to set any required variables. Env vars can be used from the tests to trigger the function itself. 

Please see https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md#method-structure.

Reply:
done